### PR TITLE
Release preparation

### PR DESCRIPTION
**Pull request by lucafavatella**
In scope of https://www.pivotaltracker.com/story/show/156954334

### CODE REVIEW

**Comment by lucafavatella** [comment]
Force-pushed with MVP -> MVPs.

File: docs/build.md [hunk 1]
```diff
@@ -61,10 +61,10 @@ Epoch source code can be obtained by cloning the public [GitHub repository](http
 git clone https://github.com/aeternity/epoch.git epoch && cd epoch
 ```
 
-Then checkout the version to be build, e.g. `v0.12.0`:
+Then checkout the version to be build, e.g. `v0.13.0`:
```
Comment (dincho) [inline docs/build.md]: Updating the docs with version is annoying, maybe we should use at least bash var, or better wildcards :)

File: docs/release-notes/RELEASE-NOTES-0.13.0.md [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
+* Adds channels functionality. This impacts consesus.
+* Changes the cryptographic signing to use Curve25519. This means all old node/signing keys are not useable anymore (`key/key.pub` is replaced by `sign_key/sign_key.pub`). This impacts consensus.
 * Enforces contiguous nonces for transactions from the same account. This impacts consensus.
 * Enforces positive transaction fee for all transactions. This impacts consensus.
 * Persists the pending transactions in the DB. This impacts the persisted DB.
-* Add channels functionality. This impacts consesus.
-* Does this. This impacts consensus.
-* Does that.
 * Improves the stability of the testnet.
+* Add much functionality to smart contracts which impacting consensus. The updates includes:
```
Comment (hanssv) [inline docs/release-notes/RELEASE-NOTES-0.13.0.md]: This sentence reads a bit badly?!

Comment (evdoks) [reply docs/release-notes/RELEASE-NOTES-0.13.0.md]: I've updated the text.

**Pull request merged**